.github/workflows/run-tests.yml

```diff
@@ -127,8 +129,17 @@ jobs:
           node-version: '18.x'
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
+      - name: Prefer npm global on windows
+        if: runner.os == 'Windows'
+        # On Windows by default PATH prefers corepack bundled with Node.js
+        # This prepends npm global to PATH to ensure that npm installed global corepack is used instead
+        run: |
+          echo "$(npm config get prefix)" >> "$GITHUB_PATH"
+        shell: bash
       - name: setup pnpm/yarn
-        run: corepack enable
+        run: |
+          npm install -g corepack --force
```

Comment:
Learning - why `corepack@latest` above, and then `--force` in this step?

Reply:
The `@latest` selector in the `e2e` is probably not needed - just result of iterating and trying to find working solution. I will remove it to keep things in sync.

The `--force` here is needed, because Windows one otherwise would fail to install - without it I was getting following error:
```
npm error code EEXIST
npm error path C:\npm\prefix\yarn
npm error EEXIST: file already exists
npm error File exists: C:\npm\prefix\yarn
npm error Remove the existing file and try again, or run npm
npm error with --force to overwrite files recklessly.
```

And we only needed it in jobs that run integration tests because only those use Windows runners. E2E ones are linux only hence don't need it.